ksqldb-common/src/main/java/io/confluent/ksql/function/KsqlScalarFunction.java

```diff
@@ -93,6 +94,7 @@ public static KsqlScalarFunction createLegacyBuiltIn(
     );
   }
 
+  @SuppressFBWarnings(value = "EI_EXPOSE_REP")
   public Class<? extends Kudf> getKudfClass() {
     return kudfClass;
```

Comment:
`Class.forName(kudfClass.getName(), true, kudfClass.getClassLoader());` might work

Reply:
Maybe, but for this case we need to add exception handling? I think we should discuss with the team (similar issue for `URL` and `URI`...)